src/test/java/net/datafaker/providers/entertainment/SeveranceTest.java

```diff
@@ -0,0 +1,23 @@
+package net.datafaker.providers.entertainment;
+
+import net.datafaker.Faker;
+
+import java.util.Collection;
+import java.util.List;
+
+class SeveranceTest extends EntertainmentFakerTest {
+
+    private final Severance futurama = getFaker().severance();
+
+    @Override
+    protected Collection<TestSpec> providerListTest() {
+        return List.of(
+            TestSpec.of(futurama::character, "severance.characters")
```

Comment:
Oops

Reply:
Oh it's because of the mixed declaration.

Reply:
Ooops, copy-paste error :)